docker-compose/.env

```diff
@@ -0,0 +1,4 @@
+DOCKER_IP=192.168.1.9

```

Comment:
Can we add comments to this file? In this case can you add the comment `Replace by your IP address`

Reply:
Ok